content/tutorials/webrtc/basics/en/index.html

```diff
@@ -74,6 +76,9 @@ <h2 id="toc-history">A very short history of WebRTC</h2>
 <p>The guiding principles of the WebRTC project are that its APIs should be open source, free, standardized, built into web browsers and more efficient than existing technologies.</p>
 
 <h2 id="toc-where">Where are we now?</h2>
+<p>WebRTC is used in various apps like Whatsapp, FaceTime messenger, appear.in and platforms such as TokBox. There is even an experimental WebRTC enabled iOS Browser named Bowser. WebRTC has also been integrated with <a href="https://labs.ericsson.com/developer-community/blog/beyond-html5-conversational-voice-and-video-implemented-webkit-gtk" title="Ericsson article about WebKitGTK+">WebKitGTK+</a> and <a href="http://www.youtube.com/watch?v=Vm5ebKWKNE8" title="basysKom showcasing WebRTC based video chat in QML application">Qt</a> native apps.</p>
+
+<p>Also Microsoft <a href="http://blogs.windows.com/msedgedev/2015/05/13/announcing-media-capture-functionality-in-microsoft-edge/" title="media capture functionality in Microsoft Edge">recently</a> added MediaCapture and Stream APIs to Edge.</p>
```

Comment:
Could you rephrase this so as not to say 'recently', maybe just 'Microsoft has added MediaCapture and Stream APIs to Edge.'